javascript-data-model/order-history.js

```diff
@@ -0,0 +1,51 @@
+var orderHistory = [
+  {
+    OrderPlaced: 'August 4, 2020',
+    Total: '$34.00',
+    ShipTo: 'JS Masher',
+    OrderNumber: '114-3941689-8772232',
+    Delivered: 'Aug 8, 2020',
+    HandedDirectlyToResident: 'Yes',
+    Title: 'JavaScript for impatient programmers',
```

Comment:
My only recommendation is to create a property `items` which is an array of objects, and then have all of the data for each product stored in individual objects. That allows for big orders with lots of items and a wide variety of item types.